rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/Participant.java

```diff
@@ -11,11 +11,7 @@
 import pl.oaza.warszawa.dor.rekolekcje.api.participants.value.HealthReportValue;
 import pl.oaza.warszawa.dor.rekolekcje.api.participants.value.PersonalDataValue;
 
-import javax.persistence.Embedded;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import javax.persistence.*;
```

Comment:
Nie używamy `*` w importach.
Zobacz sobie: [tu](https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad)
i zmień sobie ustawienia w intelliJ: [instrukcja](https://i.stack.imgur.com/J4xwT.png)

Reply:
W projekcie jest dodany plik z ustawieniami, możesz sobie zaimportować do IntelliJ:
`intellij-java-google-style.xml`